tests/unittests/BackendTestUtils.h

```diff
@@ -38,6 +45,61 @@ class MockBackend : public Backend {
   bool isOpSupported(Kinded::Kind opKind, ElemKind elementTy) const override {
     return false;
   }
+
+  bool generateInst(Node *N, IRGenVisitor &irgen) const override {
+    return false;
+  }
+};
+
+/// MockBackend2 used only for unit testing to test custom lowering from Node to
+/// Instruction IR.
+class MockBackend2 : public Backend {
```

Comment:
nit: Can you name `MockBackend2` a little more informatively? E.g. `MockBackendCustomIRGen`